worlds/blasphemous/__init__.py

```diff
@@ -0,0 +1,430 @@
+from typing import Dict, Set, Any
+from BaseClasses import Region, Entrance, Location, Item, Tutorial, ItemClassification
+from ..AutoWorld import World, WebWorld
+from .Items import base_id, item_table, group_table, tears_set, reliquary_set, skill_set
+from .Locations import location_table, shop_set
+from .Exits import region_exit_table, exit_lookup_table
+from .Rules import rules
+from worlds.generic.Rules import set_rule
+from .Options import blasphemous_options
+from . import Vanilla
+
+
+class BlasphemousWeb(WebWorld):
+    theme = "stone"
+    tutorials = [Tutorial(
+        "Multiworld Setup Guide",
+        "A guide to setting up the Blasphemous randomizer connected to an Archipelago Multiworld",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["TRPG"]
+    )]
+
+
+class BlasphemousWorld(World):
+    """
+    Blasphemous is a challenging Metroidvania set in the cursed land of Cvstodia. Play as the Penitent One, trapped
+    in an endless cycle of death and rebirth, and free the world from it's terrible fate in your quest to break
+    your eternal damnation!
+    """
+
+    game: str = "Blasphemous"
+    web = BlasphemousWeb()
+    data_version: 1
+
+    item_name_to_id = {item["name"]: (base_id + index) for index, item in enumerate(item_table)}
+    location_name_to_id = {loc["name"]: (base_id + index) for index, loc in enumerate(location_table)}
+    location_name_to_game_id = {loc["name"]: loc["game_id"] for loc in location_table}
+
+    item_name_groups = group_table
+    option_definitions = blasphemous_options
+
+
+    def set_rules(self):
+        rules(self)
+
+
+    def create_item(self, name: str) -> "BlasphemousItem":
+        item_id: int = self.item_name_to_id[name]
+        id = item_id - base_id
+
+        return BlasphemousItem(name, item_table[id]["classification"], item_id, player=self.player)
+
+
+    def create_event(self, event: str):
+        return BlasphemousItem(event, ItemClassification.progression_skip_balancing, None, self.player)
+
+
+    def get_filler_item_name(self) -> str:
+        return self.multiworld.random.choice(tears_set)
+
+
+    def generate_basic(self):
+        placed_items = []
+
+        for i in range(len(Vanilla.unrandomized_dict.values())):
+                placed_items.append(list(Vanilla.unrandomized_dict.values())[i])
```

Comment:
```suggestion
        placed_items.extend(Vanilla.unrandomized_dict.values())
```

..and all following.
Since you .count() later, I'd recommend collections.Counter()